src/EffectPlugin.h

```diff
@@ -84,6 +84,15 @@ class AUDACITY_DLL_API EffectPlugin
       std::shared_ptr<EffectInstance> &pInstance, EffectSettingsAccess &access,
       bool forceModal = false) = 0;
 
+   //! Returns the EffectUIClientInterface instance for this effect
+   /*!
+    * Usually returns self, but in case of NoiseReduction, it returns
```

Comment:
The comment is not correct -- NoiseReduction does not override the function.  And I don't think it needs to because it is not eligible for effect stacks.